preprocessing/DataCleanser.py

```diff
@@ -1,17 +1,23 @@
 import re
+import numpy as np
 
 
 def dropNulls(df_covid):
     """
-    Removes any rows that contain a null value. We print out the abstract description here because we know there are
-    null value in there. We made the abstract a None type whenever no abstract was present in a JSON file.
+    Removes any rows that contain a null value. We currently create a string 'no abstract provided' if the abstract
+    is not present, so this method is to account for potential mishandled abstract entries.  For 'body_text'
+    entries that are empty strings, we convert them to NaN and then drop them.
 
     Should be completed before running other methods to reduce the number of rows examined.
 
     :param df_covid: Dataframe of covid data.
     :return: Data frame without any null values.
     """
-    df_covid.dropna(inplace=True)
+    df_covid['abstract'].replace('', 'no abstract provided', inplace=True)
+    df_covid['body_text'].replace('', np.nan, inplace=True)
+    df_covid.dropna(subset=['body_text', 'authors'], inplace=True)
```

Comment:
This is where the rows are dropped if body_text or authors is null @redhed976

Reply:
I forgot to make authors = NaN if they were empty...
Can you fix that in your branch when you fix the whitespace issue?